buildSrc/src/main/groovy/org/opensearch/gradle/BuildPlugin.groovy

```diff
@@ -81,6 +81,7 @@ class BuildPlugin implements Plugin<Project> {
         configureLicenseAndNotice(project)
         project.pluginManager.apply('opensearch.publish')
         project.pluginManager.apply(DependenciesInfoPlugin)
+        project.pluginManager.apply('jacoco')
```

Comment:
Just curious: Where is groovy pulling jacoco dependency from?

Reply:
Finally I found one, the dependency under `buildSrc` is fetched from the repository that declared in `buildSrc/build.gradle` - https://github.com/opensearch-project/OpenSearch/blob/1.0.0/buildSrc/build.gradle#L97

Reply:
If I assuem Gradle plugins that applied to sub-projects are pulled from the repository declared in `buildSrc/build.gradle`, I don't understand why Gradle plugins in the root project couldn't be pulled there, and I need to declare repository again in root "build.gralde". 🤔 Would like to find the reason.